pkgs/top-level/python-packages.nix

```diff
@@ -2495,9 +2496,9 @@ in {
 
   jupyterlab_launcher = callPackage ../development/python-modules/jupyterlab_launcher { };
 
-  jupyterlab_server = callPackage ../development/python-modules/jupyterlab_server { };
+  jupyterlab_server = throw "pkgs.python3.pkgs.jupyterlab_server has moved to pkgs.jupyterlab_server";
```

Comment:
I think we want to keep as a python module for reasons explained in https://github.com/NixOS/nixpkgs/pull/67423#issuecomment-538420909